sql/item_strfunc.cc

```diff
@@ -1003,7 +1003,7 @@ String *Item_func_des_encrypt::val_str(String *str)
   push_warning_printf(thd, Sql_condition::WARN_LEVEL_WARN,
                       ER_FEATURE_DISABLED, ER_THD(thd, ER_FEATURE_DISABLED),
                       "des_encrypt", "--with-ssl");
```

Comment:
this becomes quite misleading. Should be rephrased, for example
```c
  push_warning_printf(thd, Sql_condition::WARN_LEVEL_WARN,
                      ER_FEATURE_DISABLED,
                     "MariaDB is linked with OpenSSL that has DES algorithms disabled");
```

Reply:
same in the des_decrypt, of course

Reply:
Updating to `-DHAVE_des`.